app/models/feedback_submission.rb

```diff
@@ -59,19 +63,22 @@ def submit
       return @errors.blank?
   end
 
+  def jira_issue(jira_config)
```

Comment:
```suggestion
  def build_jira_issue(jira_config)
```
Optional suggestion, but thought the naming might help to clarify at a glance.